modules/apps/configuration-admin/configuration-admin-web/src/main/resources/META-INF/resources/search_results.jsp

```diff
@@ -47,7 +47,9 @@ renderResponse.setTitle(LanguageUtil.get(request, "search-results"));
 	showSearch="<%= true %>"
 />
 
-<div class="container-fluid container-fluid-max-xl container-view">
+<clay:container
+	className="c-py-4"
```

Comment:
Another class-to-utility: `container-view` -> `c-py-4`. 

If we don't want `.container-view`,  then maybe we should get rid of it... do we want it?